src/mon/Monitor.cc

```diff
@@ -496,6 +504,50 @@ int Monitor::do_admin_command(
   return r;
 }
 
+int Monitor::backup(bool full)
+{
+  if(!backup_manager) {
```

Comment:
Is it possible for this to be null?  space between if and '(' (and please check every other condition in this PR).

Reply:
Test environments or admin socket while startup. I try to be rather save then sorry